lib/byron/test/integration/Test/Integration/Byron/Scenario/API/Transactions.hs

```diff
@@ -847,6 +847,14 @@ scenario_TRANS_REG_1670 fixture = it title $ \ctx -> do
         Empty
 
     -- ASSERTIONS
+    let transaction = head $ getFromResponse id rTxs
```

Comment:
@piotr-iohk There's no guarantee that the first tx is the one we are interested in here :thinking:
It'll be safer to simply run this check for every transaction present in the wallet, it should hold anyway.

Reply:
Added verification for all outgoing transactions -> https://github.com/input-output-hk/cardano-wallet/pull/1697/commits/f8bcfe23df814fb3d6769e2e27984c26b523cba3. Incoming txs not necessarily have input amounts.